### ISSUE TITLE
PR #353 breaks the code in my testing

### ISSUE THREAD

**Issue by maugustosilva** [keylime/keylime#359]
Prior PR #353, here is how `keylime_tenant` behaves:
```
root@css-flex14vm1:/# /usr/local/bin/keylime_tenant -c status -v 100.64.14.13 -t 100.64.14.16
Using config file /etc/keylime.conf
2020-09-22 16:51:58.888 - keylime.tenant - INFO - Setting up client TLS in /var/lib/keylime/cv_ca
2020-09-22 16:51:58.891 - keylime.tenant - INFO - Setting up client TLS in /var/lib/keylime/cv_ca
2020-09-22 16:51:58.894 - keylime.tenant - WARNING - Using default UUID D432FBB3-D2F1-4A97-9EF7-75BD81C00000
AQUI
2020-09-22 16:51:58.894 - keylime.httpclient_requests - DEBUG - Making HTTP GET request to 100.64.14.13:8881/agents/D432FBB3-D2F1-4A97-9EF7-75BD81C00000
2020-09-22 16:51:58.935 - keylime.tenant - ERROR - Agent D432FBB3-D2F1-4A97-9EF7-75BD81C00000 does not exist on the verifier. Please try to add or update agent
```

Afterwards, here is what I see:

```
root@css-flex14vm1:/KL/keylime# /usr/local/bin/keylime_tenant -c status -v 100.64.14.13 -t 100.64.14.16
Using config file /etc/keylime.conf
2020-09-22 16:52:21.031 - keylime.tenant - INFO - Setting up client TLS in /var/lib/keylime/cv_ca
2020-09-22 16:52:21.034 - keylime.tenant - INFO - Setting up client TLS in /var/lib/keylime/cv_ca
2020-09-22 16:52:21.037 - keylime.tenant - WARNING - Using default UUID D432FBB3-D2F1-4A97-9EF7-75BD81C00000
2020-09-22 16:52:21.037 - keylime.httpclient_requests - DEBUG - Making HTTP GET request to 100.64.14.13:8881/agents/D432FBB3-D2F1-4A97-9EF7-75BD81C00000
2020-09-22 16:52:21.042 - keylime.httpclient_requests - ERROR - ConnectionError: [Errno 104] Connection reset by peer
2020-09-22 16:52:21.042 - keylime.tenant - ERROR - Cannot connect to Verifier at 100.64.14.13 with Port 8881. Connection refused.
```

The problem seems to be a simple one...

This new `max_buffer_size` parameter added to tornado HTTP server requires and integer argument, and we're passing a string. Instead of simply failing, as one would normally expect, the server starts, but the server (silently) refuses all connections.

The fix is rather simple, just change the code in `keylime/cloud_verifier_tornado.py` (line 672)
```
        max_upload_size = int(config.get('cloud_verifier', 'max_upload_size'))
```

and we can the keep line 712 as is:

```
     server = tornado.httpserver.HTTPServer(app, ssl_options=context, max_buffer_size=max_upload_size)
```

**Comment by maugustosilva** [comment]
@mpeters do you agree?

**Comment by mpeters** [comment]
This is very strange. It definitely works for me as a string. But it also keeps working as an int, so I'm fine with the change (it also makes a non-int in the config fail in a better way, so that's an added bonus).

**Comment by maugustosilva** [comment]
@mpeters @lukehinds sorry, the fix is still broken for me...

```
  File "/usr/local/lib/python3.6/dist-packages/keylime-0.0.0-py3.6.egg/keylime/cloud_verifier_tornado.py", line 672, in main
    int(max_upload_size = config.get('cloud_verifier', 'max_upload_size'))
TypeError: 'max_upload_size' is an invalid keyword argument for this function
```

This is an `Ubuntu 18.04` container with `python 3.6.9` btw

**Comment by galmasi** [comment]
@mpeters can you please look up the exact version of tornado in your work environment? I imagine it is a Fedora package? but if so we need to know what version of tornado. Marcio's code uses pip. 

I'm going out on a limb here but I expect the problem is due to tornado version divergence.

**Comment by maugustosilva** [comment]
Just to be clear, there are now **two** problems:
1) Something (maybe different versions of `tornado`)  are causing a silent failure in case of `max_buffer_size` is not an integer
2) This particular construction for the fix (PR #360) seems to be causing a problem on my `Ubuntu 18.04` with `python 3.6.9`

I believe my initial suggestion -  `max_upload_size = int(config.get('cloud_verifier', 'max_upload_size'))` - is the one that seems to work to the broadest base of versions.

**Comment by mpeters** [comment]
@maugustosilva I agree. New commit that should fix this for all cases: https://github.com/keylime/keylime/pull/363

**Comment by maugustosilva** [comment]
I can confirm that this has been fixed with PR #363. Closing it now
